src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestCaseDTO.java

```diff
@@ -33,4 +37,45 @@ public TestCaseDTO(String name, String classname, double time) {
     public boolean isSuccessful() {
         return CollectionUtils.isEmpty(errors) && CollectionUtils.isEmpty(failures);
     }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public List<String> getMessage() {
+        return extractMessageFromTestCase().map(Collections::singletonList).orElse(Collections.emptyList());
+    }
+
+    private Optional<String> extractMessageFromTestCase() {
```

Comment:
We moved this method inside a TestCase-class so we no longer need the `fromTestCase` part in the method name.
```suggestion
    public List<String> getMessage() {
        return extractMessage().map(Collections::singletonList).orElse(Collections.emptyList());
    }

    private Optional<String> extractMessage() {
```